From f91a5adf7e8d778f8f006934400249ec9c84717c Mon Sep 17 00:00:00 2001
From: Felix Fietkau <nbd@openwrt.org>
Date: Fri, 2 Nov 2012 16:23:47 +0530
Subject: [PATCH] ath9k_hw: fix signal strength / noise measurements on ar9462

The nominal noise floor for most channels is -120, though on some it can
reach -127. Use -120 for converting noise floor to channel noise to
avoid overestimating rx signal strength and channel noise.

Signed-off-by: Felix Fietkau <nbd@openwrt.org>
---
 drivers/net/wireless/ath/ath9k/ar9003_phy.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/net/wireless/ath/ath9k/ar9003_phy.h
+++ b/drivers/net/wireless/ath/ath9k/ar9003_phy.h
@@ -344,9 +344,9 @@
 #define AR_PHY_CCA_MAX_GOOD_VAL_9300_2GHZ     -95
 #define AR_PHY_CCA_MAX_GOOD_VAL_9300_5GHZ     -100
 
-#define AR_PHY_CCA_NOM_VAL_9462_2GHZ          -127
+#define AR_PHY_CCA_NOM_VAL_9462_2GHZ          -120
 #define AR_PHY_CCA_MIN_GOOD_VAL_9462_2GHZ     -127
-#define AR_PHY_CCA_NOM_VAL_9462_5GHZ          -127
+#define AR_PHY_CCA_NOM_VAL_9462_5GHZ          -120
 #define AR_PHY_CCA_MIN_GOOD_VAL_9462_5GHZ     -127
 
 #define AR_PHY_CCA_NOM_VAL_9330_2GHZ          -118
